### DISCUSSION TITLE
Google Colab crashes with graph classification tutorial

### DISCUSSION THREAD

**Discussion by paulamartingonzalez** [pyg-team/pytorch_geometric#2936] [Q&A]
I am trying to run the [graph classification tutorial](https://colab.research.google.com/drive/1I8a0DfQ3fI7Njc62__mVXUlcAleUclnb) with a custom dataset and google Colab crashes when training. 

It does work well when I run the tutorial but my dataset is not huge as is not in memory either so I wasn't expecting this to be a problem.

The log I get is "src/tcmalloc.cc:283] Attempt to free invalid pointer 0x7ffdafbc10d8" before I get "KernelRestarter: restarting kernel (1/5), keep random ports" and "WARNING:root:kernel 7180f956-fbf9-4e1b-adb7-617812dd4ee1 restarted".

Do you have any hint of what might be happening? Any suggestion is welcomed!

**Comment by rusty1s** [comment]
I haven't seen this error before, sorry. Which part of the script causes this issue?

**Reply by paulamartingonzalez** [reply]
Hi @rusty1s , after a little bit of digging I realised my custom training dataset had a small bug (one graph less than expected on the files inputed). I just leave this here for future reference if needed, but it is solved and the code is working. Thanks so much!

**Reply by mburaksayici** [reply]
Hi Paula, I'm having exactly the same error, can you lil bit elaborate how did you solve?

**Reply by paulamartingonzalez** [reply]
Doing a bit of google search, the error could be different things (mismatched dimensions, wrong loss function...). In my case, there was a graph missing (but expected) in the custom dataset so I had to clean the database a bit more before doing the dataset. Hope this helps!
